### PR TITLE
Update GEP-03 for new reference style

### PR DESCRIPTION
**Pull request by mjbloemer**
This PR [updates](https://github.com/iza-institute-of-labor-economics/gettsim/blob/main/docs/geps/gep-00.md#maintenance) GEP-03 to reflect changes in the citation format of the Bundesgesetzblatt (BGBl). Starting in 2023, the BGBl is published exclusively online at [recht.bund.de](https://www.recht.bund.de/), and new references now use unique document numbers instead of page numbers. This update ensures consistency with the new citation style.

### CODE REVIEW

**Review by ChristianZimpelmann — APPROVED** [review]

File: docs/geps/gep-03.md [hunk 1]
```diff
@@ -167,11 +169,15 @@ These work differently and they are
 - cites the law (Gesetz "G. v."), decree (Verordnung "V. v.") or proclamation
   (Bekanntmachung "B. v.") that changes the law
 - uses German language
-- follows the style `Artikel [n] [G./V./B.] v. [DD.MM.YYYY] BGBl. I S. [SSSS].`
+- follows the style
+  - `Artikel [n] [G./V./B.] v. [DD.MM.YYYY] BGBl. I S. [SSSS].` for older references
+    published until 2022
+    - the page should be the first page of the law/decree/proclamation, not the exact
```
Comment (ChristianZimpelmann) [inline docs/geps/gep-03.md:175]: I would rather put this in brackets instead of second level bullet point.

**Review by ChristianZimpelmann — APPROVED** [review]
Except of one suggestions, nothing to complain. Thanks!

**Comment by ChristianZimpelmann** [comment]
I put the comment in brackets as I suggested.

Now, `timeout waiting for merge ref` appeared.

I think, we can merge anyway, but maybe @hmgaudecker can double-check.

**Review by hmgaudecker — APPROVED** [review]
Thank you, very good catch. 

I just wonder whether we should be going through all stuff that has changed since 2023 and adjust the refs? Could be another PR though or maybe not necessary. Feel free to merge in any case!

**Comment by MImmesberger** [comment]
> going through all stuff that has changed since 2023 and adjust the refs?

Will do that soon, then we can merge.

**Comment by MImmesberger** [comment]
Turns out all references are correct!

**Pull request merged**